### ISSUE TITLE
Docker Sentry - "There was an error loading data."

### ISSUE THREAD

**Issue by be-janshair** [getsentry/sentry#5132]
I'm using Sentry.io inside a [Docker container](https://hub.docker.com/_/sentry/) and I build the container with all the commands specified in the Docker Image description. I used PostgreSQL:latest, Redis:Latest and Sentry:latest image pull from the Docker hub.

The problem occurs when I create an issue and navigate to the detail view of the error. It says **There was an error loading data.** I checked out the container's state I have 5 containers running. 3 sentry, 1 postgres and 1 redis. Everything works fine but can't see the detail of the error.

Any relevant information will be helpful. :)

**Comment by mattrobenolt** [comment]
Check your logs from both the server and your browser. Other than that, not sure, but likely a configuration issue.

**Comment by be-janshair** [comment]
I tried on 2 different locations. 1 in an Azure VM and 2 in an AWS VM. I followed the steps in the official Sentry docker image documentations. Both has 5 containers backing the Sentry including Redis and Postgres with the same names. Everything works perfect in Azure Ubuntu VM but goes wrong in AWS EC2 instance. When I browse to the application, it says:

`Failed to load resource: the server responded with a status of 404 (NOT FOUND)`

In the browser's console.
